cmd/agent/main.go

```diff
@@ -91,11 +97,24 @@ func main() {
 	if *serverAddr == "" {
 		logrus.Fatal("please, specify the target host user -connect host:port")
 	}
-	host, _, err := net.SplitHostPort(*serverAddr)
-	if err != nil {
-		logrus.Fatal("invalid connect address, please use host:port")
+
```

Comment:
Should use "net/url" instead of manual parsing: https://gobyexample.com/url-parsing

Reply:
Done in last commit